### PR TITLE
[PTX-4584] reduce the number of cat /proc/mounts in schedops.validat…

### PR DESCRIPTION
**Pull request by JimOuPure**
…eMountsInPods

schedops.validateMountsInPods tries to validate multiple mount paths, and for each path,
it calls “cat /proc/mounts" once, that’s unnecessary

tested on http://jenkins.pwx.dev.purestorage.com/view/Control%20Plane/job/Torpedo/job/tp-basic-sharedv4-svc/72/

### CODE REVIEW

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

File: drivers/volume/portworx/schedops/k8s-schedops.go [hunk 1]
```diff
@@ -311,19 +311,18 @@ PodLoop:
 		containerPaths := getContainerPVCMountMap(*pod)
 		skipHostMountCheck := false
 		for containerName, paths := range containerPaths {
+			output, err := k8sCore.RunCommandInPod([]string{"cat", "/proc/mounts"}, pod.Name, containerName, pod.Namespace)
+			mounts := strings.Split(output, "\n")
```
Comment (pureneelesh) [inline drivers/volume/portworx/schedops/k8s-schedops.go:315]: nit: Why did you move this line before checking the err? Makes more sense to do this only if the command was successful.

**Review by pureneelesh — APPROVED** [review]

File: drivers/volume/portworx/schedops/k8s-schedops.go (hunk 1 above)
Comment (JimOuPure) [reply drivers/volume/portworx/schedops/k8s-schedops.go:315]: good point! didn't do it intentionally, let me change it
